internal/cmd/scraper/scraper.go

```diff
@@ -311,11 +290,21 @@ func Run(cfg *Config) error {
 				return errors.Wrap(err, "could not create new TelemetryEmitter")
 			}
 			emitters = append(emitters, emitter)
+		case "infra-sdk":
```

Comment:
do we need to keep "stdout" emitter? i think that @paologallinaharbur mentioned that was not working since the struct fields were not exporter.

Reply:
When using it the output is `{"","","","",""}`, however it is used to run test. 
I would keep it, even if the same is quite confusing (mock emitter would be better I guess)